lua/anywise_reg/keybinds.lua

```diff
@@ -48,36 +48,26 @@ local function setup_yank_keymaps(prefix)
     end
 end
 
-local function setup_paste_keymaps(prefix)
-    local lhs = prefix..config.paste_key
-    local rhs = '<Cmd>lua require("anywise_reg.handlers").handle_paste(\''..prefix..'\')<CR>'
-    set_keymap(lhs, rhs)
-end
-
-local function setup_paste_behind_keymaps(prefix)
-    local lhs = prefix .. config.paste_behind_key
-    local rhs =
-        '<Cmd>lua require("anywise_reg.handlers").handle_paste_behind(\'' ..
-            prefix .. '\')<CR>'
+local function setup_paste_keymaps(prefix, key, operator)
+    local lhs = prefix .. key
+    local rhs = '<Cmd>lua require("anywise_reg.handlers").handle_paste(\'' ..
+                    prefix .. '\',\'' .. operator .. '\')<CR>'
     set_keymap(lhs, rhs)
 end
 
 M.setup_keymaps = function()
     local prefixes = {'', '""'}
-    if config.paste_key ~= nil or config.paste_behind_key ~= nil then
+    if #config.paste_keys >= 1 then
```

Comment:
I think we should still register the action keymaps even if the user doesn't want the paste ones. The user might set the paste keys themselves for some reason

Reply:
Ah I just based that on your repo has 

```
    if config.paste_key ~= nil then
```

Reply:
Ah yeah, I think that was wrong :)